environments/flow/recommended.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Js-coding-standards
+ *
+ * @author      Robert Rossmann <rr.rossmann@me.com>
+ * @copyright   2016 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  plugins: ['flowtype'],
+
+  rules: {
+    // Marks Flow type identifiers as defined.
+    // Used to suppress no-undef reporting of type identifiers.
+    'flowtype/define-flow-type': 'warn',
+
+    // Enforces consistent use of trailing commas in Object and Tuple annotations.
+    // This rule takes one argument which mirrors ESLint's default comma-dangle rule.
+    'flowtype/delimiter-dangle': ['warn', 'always-multiline'],
+
+    // Enforces consistent spacing within generic type annotation parameters.
+    'flowtype/generic-spacing': ['error', 'always'],
+
+    // Checks for duplicate properties in Object annotations.
+    'flowtype/no-dupe-keys': 'error',
+
+    // Disallows Flow type imports, exports, aliases, and annotations in files missing a valid
+    // Flow file declaration (or a @noflow annotation).
+    'flowtype/no-types-missing-file-annotation': 'error',
+
+    // Enforces consistent separators between properties in Flow object types.
+    'flowtype/object-type-delimiter': ['error', 'comma'],
+
+    // Requires that all function parameters have type annotations.
+    'flowtype/require-parameter-type': [
+      'error',
+      {
+        excludeArrowFunctions: 'expressionsOnly',
+      },
+    ],
+
+    // Requires that functions have return type annotation.
+    'flowtype/require-return-type': [2, 'always'],
+
+    // Requires that all variable declarators have type annotations.
+    'flowtype/require-variable-type': 'error',
+
+    // Enforces consistent use of semicolons after type aliases.
+    'flowtype/semi': ['error', 'never'],
+
+    // Enforces consistent spacing after the type annotation colon.
+    'flowtype/space-after-type-colon': ['error', 'always'],
+
+    // Enforces consistent spacing before the opening < of generic type annotation parameters.
+    'flowtype/space-before-generic-bracket': ['error', 'always'],
```

Comment:
coding-style -> `warn`

Also I think the space before generic annotation is rarely used in other languages, so please switch to `never`.